packages/client-app/src/views/Request/RequestSection/RequestAuth.vue

```diff
@@ -184,7 +184,7 @@ const updateScheme: UpdateScheme = (path, value) =>
             <RequestAuthDataTableInput
               id="http-basic-password"
               :modelValue="activeSecurityScheme.scheme.secondValue"
-              placeholder="Password"
+              placeholder="xxxxxx"
```

Comment:
maybe?
```suggestion
              placeholder="••••••"
```

Reply:
Ya I tried this but is it too close to an actual value?

Reply:
yes not wrong, color is dimmed down, maybe we should stick to password?

Reply:
My issue with password is it's the same as the label so doesn't really help. The other thing we could do is a fake password like 123ABC